mac/scripts/build.sh

```diff
@@ -180,6 +181,19 @@ rm -f "$APP_PATH/Contents/Resources/Local.xcconfig"
 rm -rf "$APP_PATH/Contents/Resources/web/public/tests"
 echo "✓ Removed development files from bundle"
 
+# Apply ad-hoc signature for local builds (if not CI and not signing with cert)
```

Comment:
IIRC: if you sign ad-hoc, permissions will not persist across app runs. Needs to b e a real code signature.